package.json

```diff
@@ -42,6 +42,7 @@
     "test": "npm run test:unit && npm run test:e2e:smoke",
     "test:unit": "npm run test:main && npm run test:preload && npm run test:preload-docker-extension && npm run test:renderer && npm run test:tools && npm run test:extensions",
     "test:e2e:smoke": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build && xvfb-maybe vitest run tests/src/*smoke.spec.ts --threads false",
+    "test:e2e:smoke:binary": "xvfb-maybe vitest run tests/src/*smoke.spec.ts --threads false",
```

Comment:
probably two options here

- rename this line with `test:e2e:smoke:run` and add a call to this line in `test:e2e:smoke` to avoid copy/pasting the lines between the two

- change the logic in `test:e2e:smoke` to be if PODMAN_DESKTOP_BINARY is not set, run the `npm run build` task else just do noop and then run smoke tests

Reply:
going for the first option, so I do not have to handle cross platform env. var processing in npm script. thanks for advice!